package.json

```diff
@@ -23,13 +23,13 @@
   },
   "homepage": "https://github.com/getsentry/sentry-fullstory#readme",
   "peerDependencies": {
-    "@fullstorydev/browser": ">=2.0.0",
+    "@fullstory/browser": ">=1.0.0",
     "@sentry/browser": ">=4.0.0"
   },
   "dependencies": {},
   "devDependencies": {
     "@babel/core": "^7.7.2",
-    "@fullstorydev/browser": "^2.0.0",
+    "@fullstory/browser": "^1.0.0",
```

Comment:
@scefali I noticed this `devDependancy` and updated it. Is devDep requried?

Reply:
@patrick-fs We use `devDependencies` so dependencies resolve when testing locally and writing the code. So yes, we need it.